.github/workflows/ci.yml

```diff
@@ -584,9 +586,8 @@ jobs:
         uses: ./.github/actions/setup-sentry
         with:
           workdir: sentry
-          snuba: true
-          kafka: true
-          symbolicator: true
+          use-new-devservices: true
+          mode: symbolicator
```

Comment:
So we want postgres, snuba, and symbolicator running?

Reply:
That is correct, also redis

Reply:
By proxy, yeah